main.tf

```diff
@@ -12,12 +12,77 @@ provider "cloudflare" {
   where we track which version of a docker image has been deployed to an enviroment.
 */
 locals {
-  omni_staging_rpc = "http://staging.omni.network:8545"
   omni_staging_ws = "ws://staging.omni.network:8546"
+  omni_chain_config_staging = {
+    rpc_addr = "http://staging.omni.network:8545"
+    default_start_block = 0
+    confirmation_block_count = 0
+    syncInterval = 10
+    portal_addr = "0x1212400000000000000000000000000000000001"
```

Comment:
would rename this to omni_predeploy_addr in an upcoming issue, to minimize changes